dts/bindings/power-domain/intel,power-domain.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: Power domains for intel ADSP
+
+compatible: "intel,power-domain"
+
```

Comment:
this is not compatible with what is being used in `power_domain_intel_adsp.c`.

`#define DT_DRV_COMPAT intel_adsp_power_domain`

Reply:
ditto